packages/components/psammead-grid/src/index.jsx

```diff
@@ -130,8 +130,7 @@ const gridFallbacks = css`
         styles: `
           display: inline-block;
           width: ${(100 * columns[group]) / parentColumns[group]}%;
-          vertical-align: top; `,
-        /* [1] */
```

Comment:
Could you keep this comment? It is a reference to:
```
/*
 * 1 We vertically align to the top so that sibling
 *   grid items that are placed side-by-side within a row
 *   have their text and images aligned
 */
```